client/utils/getCookie.js

```diff
@@ -1,9 +1,7 @@
-const getCookies = () => {
+export const getCookies = (cookie) => {
   const arr = document.cookie.split('; ')
```

Comment:
@AStashevskaya Found some wrong behaviour, when you are checking `el.startsWith(cookie)` you could find wrong cookie, which starts with same name. Example:

<img width="273" alt="image" src="https://user-images.githubusercontent.com/12561302/118830894-9dc30480-b8bf-11eb-8c43-43baa72d067c.png">

Better to check, that `el.startsWith(cookie + '=')`.

Also you can avoid second `arr1.find`, because arr` it should contain only 2 elements: `const [cookieName, cookieValue] = currentCookie.split('=')`